### DISCUSSION TITLE
Is it possible to have command line completion in the terminal?

### DISCUSSION THREAD

**Discussion by tmigone** [coder/code-server#2085] [Q&A] [answered]
Currently if I hit `TAB` it tabs to the right as if I entered the `TAB` char instead of doing command line completion.
This is on chrome/macOS.

**Comment by tmigone** [comment, marked as answer]
Got it, need to change default terminal to `bash` instead of `sh`.

**Reply by clint456** [reply]
I have same problem and think you are right,but how to change from sh to bash🫨

**Reply by clint456** [reply]
![image](https://github.com/coder/code-server/assets/83180587/604c3064-f337-42bf-bbb9-d465df0f7ac2)

- you see,my terminal is `bash`,but the problem is still likely your description.

**Reply by clint456** [reply]
okay,i known how to do this,Thanks. 
[someone has the same confuses,click here!](https://github.com/coder/code-server/discussions/6350)
